test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv

```diff
@@ -1,98 +1,98 @@
 TopicTitle,HelpURI
 Add-ContentFileSystem,https://go.microsoft.com/fwlink/?LinkID=204573
 Add-JobTrigger,https://go.microsoft.com/fwlink/?LinkID=223913
-Connect-PSSession,https://go.microsoft.com/fwlink/?LinkID=210604
-ConvertFrom-Json,https://go.microsoft.com/fwlink/?LinkID=217031
-ConvertTo-Json,https://go.microsoft.com/fwlink/?LinkID=217032
+Add-LocalGroupMember,https://go.microsoft.com/fwlink/?LinkId=717987
+Clear-RecycleBin,https://go.microsoft.com/fwlink/?LinkId=2109377
+Compress-Archive,https://go.microsoft.com/fwlink/?linkid=2096473
+Connect-PSSession,https://go.microsoft.com/fwlink/?linkid=2096694
+ConvertFrom-Json,https://go.microsoft.com/fwlink/?linkid=2096606
+ConvertFrom-Markdown,https://go.microsoft.com/fwlink/?linkid=2006503
+ConvertFrom-String,https://go.microsoft.com/fwlink/?LinkId=507579
+Convert-String,https://go.microsoft.com/fwlink/?LinkId=528577
+ConvertTo-Json,https://go.microsoft.com/fwlink/?linkid=2096925
+Debug-Runspace,https://go.microsoft.com/fwlink/?linkid=2096917
 Disable-JobTrigger,https://go.microsoft.com/fwlink/?LinkID=223918
+Disable-LocalUser,https://go.microsoft.com/fwlink/?LinkId=717986
+Disable-RunspaceDebug,https://go.microsoft.com/fwlink/?linkid=2096924
 Disable-ScheduledJob,https://go.microsoft.com/fwlink/?LinkID=223927
-Disconnect-PSSession,https://go.microsoft.com/fwlink/?LinkID=210605
+Disconnect-PSSession,https://go.microsoft.com/fwlink/?linkid=2096576
 Enable-JobTrigger,https://go.microsoft.com/fwlink/?LinkID=223917
+Enable-LocalUser,https://go.microsoft.com/fwlink/?LinkId=717985
+Enable-RunspaceDebug,https://go.microsoft.com/fwlink/?linkid=2096831
 Enable-ScheduledJob,https://go.microsoft.com/fwlink/?LinkID=223926
+Enter-PSHostProcess,https://go.microsoft.com/fwlink/?linkid=2096580
+Exit-PSHostProcess,https://go.microsoft.com/fwlink/?linkid=2096583
+Expand-Archive,https://go.microsoft.com/fwlink/?linkid=2096769
```

Comment:
Please remove this change till the module has shipped with the update.

Reply:
Removed `Expand-Archive` from the CSV file.